exercises/book-store/description.md

```diff
@@ -60,8 +60,8 @@ This would give a total of:
 
 Resulting in:
 
-- 4 * (8 - 1.60) == 4 * 6.40 == $25.60
-- +4 * (8 - 1.60) == 4 * 6.40 == $25.60
+- 4 \* (8 - 1.60) == 4 \* 6.40 == $25.60
```

Comment:
Same here, regarding escaping the `*` in lieu of making this a code block.

Reply:
Since the dollars signs are here, I tend to "mathematic expression" more than "code expression".  And perhaps this change is not meant for this PR given the title of the PR.